index.html

```diff
@@ -0,0 +1,288 @@
+	<div class="col-3">
+		<button class="main_btn goBack">Go back</button>
+	</div>
+	<div class="col-3">
+		<p class="logo"><img src="css/img/kottans.png"></p>
+	</div>
+	<div class="col-3 sign">
+		<button class="main_btn btn_sgn">Sign In</button>
+	</div>
+</header>
+
+
+<section class="yourOrder">
+	<div class="col-6">
+		<div class="details">
+			<div class="blank"><!--<p><img src="https://png.icons8.com/material/26/6c748d/file.png"></p>--> <p>Your order</p></div>
+			<div>
+				<ul class="list">
+					<li><span>Items</span></br></br>2 documents</li>
+					<li><span>Details</span></br></br>2 languages</li>
+					<li><span>Price</span></br></br>$ 696.06</li>
+					<li><span>Delivery</span></br></br>48 hours*</li>
+				</ul>
+			</div>
+		</div>
+		<div class="detBut">
+		<button class="act_btn">Place order</button>
+		<button class="main_btn print">Print a Quote</button>
+		</div>
+	</div>
+	<div class="col-3">
+		<div class="help1"><!--<p><img src="https://png.icons8.com/windows/26/767d94/lifebuoy.png"></p>--><p>Help center</p></div>
+		<div class="help">
+			<p>If you need assistance while placing your order, contact one of our beloved Project Manager by phone or email.</p>
+			<p>+971(0) 555 555</p> <p>help@qtest.com</p>
+		</div>
+	</div>
+</section>
+<form class="main">
+	<section class="select">
+		<h3>Select what you want to translate</h3>
+			<div class="form">
+				<label for="website">
+					<input hidden type="radio" name="radio1" id="website" value="website">
+						<div class="select-wrapper">
+							<img src="css/img/website.png">
+						</div>
+				</label>
+				<label for="document">
+					<input hidden type="radio" name="radio1" id="document" value="document">
+						<div class="select-wrapper">
+							<img src="css/img/documents.png">
+						</div>
+				</label>
+				<label for="social">
+					<input hidden type="radio" name="radio1" id="social" value="social">
+						<div class="select-wrapper">
+							<img src="css/img/social.png">
+						</div>	
+				</label>
+			</div>
+	</section>
+	<section class="one">
+		<div class="upload">
+			<p class="cirkle">1</p>
+			<h3>Upload your documents</h3>
+			<p class="types">(Doc, Docx, Text, PDF, Markdown)</p>
+
+		</div>
+			<button class="main_btn upload_btn">Upload a new file</button>
+		<table>
+			<tr>
+				<th>File name</th>
+				<th>Flie size</th>
+				<th>Number of words</th>
+				<th>Remove</th>
+			</tr>
+			<tr>
+				<td class="doc">business_operations.docx</td>
+				<td>78kb</td>
+				<td>842</td>
+				<td class="trash"></td>
+			</tr>
+			<tr>
+				<td class="doc">business_financial.docx</td>
+				<td>26kb</td>
+				<td>2,326</td>
+				<td class="trash"></td>
+			</tr>
+		</table>
+	</section>
+	<section class="two">
+		<div class="upload">
+			<p class="cirkle2">2</p>
+			<h3>Choose youre languages pairings</h3>
+		</div>
+		<div class="language">
+			<div class="from">
+			<p>From</p>
+			<div class="form_select">
+				<select type="text" name="capacity" required="">
```

Comment:
`required` is boolean - it is true if it is present. so, just `required` is enough